src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -1542,8 +1553,17 @@ static int xclmgmt_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 		xocl_xmc_get_serial_num(lro);
 
 	(void) xocl_hwmon_sdm_get_sensors_list(lro, true);
-
 	xocl_drvinst_set_offline(lro, false);
+
+	(void) xclmgmt_check_device_ready(lro);
```

Comment:
Does this function returns the vmr status?  (void) does not look right.

Reply:
It returns an error code detailing the health of the device. After speaking with Stephen and David we decided to not allow the driver to dictate device condition to the tool. Instead the function call outputs an dmesg error in the event an issue is found.